### PR TITLE
Update documentation regarding BOTS file and parameter handling

### PR DESCRIPTION
**Pull request by schacht-certat**
- DOC: Remove the BOTS file from the documentation
- DOC: update documentation regarding the parameters

### CODE REVIEW

File: docs/dev/guide.rst [hunk 1]
```diff
@@ -744,18 +746,12 @@ When calling the file directly, only the tests in this file for the bot will be
...
-In the end, the new information about the new bot should be added to BOTS file
-located at `intelmq/bots`. Note that the file is sorted!
-
 Cache
 =====
 
 Bots can use a Redis database as cache instance. Use the `intelmq.lib.utils.Cache` class to set this up and/or look at existing bots, like the `cymru_whois` expert how the cache can be used.
 Bots must set a TTL for all keys that are cached to avoid caches growing endless over time.
-Bots must use the Redis databases `>=` 10, but not those already used by other bots. See `bots/BOTS` what databases are already used.
+Bots must use the Redis databases `>=` 10, but not those already used by other bots. Look at `intelmqctl list bots` to see which databases are already used.
```
Comment (wagner-certat) [inline docs/dev/guide.rst:754]: This command does not show the databases. Maybe recommend `find intelmq -type f -name '*.py' -exec grep -r 'redis_cache_db' {} \+` (not good, but it works).

File: docs/user/configuration-management.rst [hunk 2]
```diff
@@ -54,9 +54,8 @@ For new installations a default setup with some examples is provided by the `int
 * ``defaults.conf``: default values for all bots and their behavior, e.g. error handling, log options and pipeline configuration. Will be :issue:`removed in the future <267>`.
 * ``runtime.conf``: Configuration for the individual bots. See :doc:`bots` for more details.
 * ``pipeline.conf``: Defines source and destination queues per bot (i.e. where does a bot get its data from, where does it send it to?).
-* ``BOTS``: Includes configuration hints for all bots. E.g. feed URLs or database connection parameters. Use this as a template for ``runtime.conf``. This is also read by the intelmq-manager.
 
-To configure a new bot, you need to define and configure it in ``runtime.conf`` using the template from BOTS.
+To configure a new bot, you need to define and configure it in ``runtime.conf``. You can base your configuration on the outputp of ``intelmqctl list bots``.
```
Comment (wagner-certat) [inline docs/user/configuration-management.rst:58]: typo `outputp`

File: docs/user/configuration-management.rst [hunk 3]
```diff
@@ -336,7 +335,7 @@ Setting the statistics (and cache) parameters is necessary when the local redis
 Runtime Configuration
 *********************
 
-This configuration is used by each bot to load its specific (runtime) parameters. Usually, the ``BOTS`` file is used to generate ``runtime.conf``. Also, the IntelMQ Manager generates this configuration. You may edit it manually as well. Be sure to re-load the bot (see the :doc:`intelmqctl`).
+This configuration is used by each bot to load its specific (runtime) parameters. You can use the output of ``intelmqctl list bots`` to write the ``runtime.conf``. Also, the IntelMQ Manager generates this configuration. You may edit it manually as well. Be sure to re-load the bot (see the :doc:`intelmqctl`).
```
Comment (wagner-certat) [inline docs/user/configuration-management.rst:338]: `intelmqctl -t josn list bots`? Actually I'm missing a reference to the feeds docs here, but that could be out of scope here. (and a reference to `contrib/feeds-config-generator` could be added as well).

**Review by wagner-certat — CHANGES_REQUESTED** [review]

File: docs/dev/guide.rst (hunk 1 above)
Comment (schacht-certat) [reply docs/dev/guide.rst]: I've used the proposed command. Its the developer docs, so I think its okey for now.

File: docs/user/configuration-management.rst (hunk 2 above)
Comment (schacht-certat) [reply docs/user/configuration-management.rst]: fixed

File: docs/user/configuration-management.rst (hunk 3 above)
Comment (schacht-certat) [reply docs/user/configuration-management.rst]: fixed. not sure what reference you're referring to or what you have in mind, but if you create a separate issue, i'll have a look

**Review by wagner-certat — APPROVED** [review]

**Review by waldbauer-certat — APPROVED** [review]

**Pull request merged**